packages/utils/src/TestUtils.ts

```diff
@@ -1,5 +1,28 @@
+interface MockContext {
+  arc: () => void;
+  beginPath: () => void;
+  clip: () => void;
+  closePath: () => void;
+  createLinearGradient: () => void;
```

Comment:
I think this should be `createLinearGradient: jest.Mock<{ addColorStop: jest.Mock<void> }>`